data/transit/route/bus.json

```diff
@@ -172,17 +181,6 @@
         "route": "bus"
       }
     },
-    {
-      "displayName": "AIM Mobilità",
```

Comment:
![图片](https://user-images.githubusercontent.com/42690037/210770921-37e9bd7a-f706-41d2-ba6e-b124f5cbe134.png)

There are still 400+ elements using this nerwork name, it will be collected again if you only delete this entry. Did it change its name, was it bought by other company, or did it have to be removed for some other reason?

Reply:
Sorry if I answer only now.
The AIM company managed a lot of public services in Vicenza (Italy) (managed garbage disposal, transport, cemetery services, street lamp management etc...), now the public transport part has dissolved from AIM and has been created the new company SVT